Modify forking layout for Mega online event (#134)
coderefinery/git-collaborative: _episodes/03-distributed.md

### Part A: Fork and clone

In the **groups/breakout-rooms**, one person like the helper, do a necessary preparatory step. The person creates a repository

> from a template in her or his GitHub user space. We do not use the templates directly but

Reviewer: ```suggestion
from a template in their GitHub user space. We do not use the templates directly but
```
